app/Resources/views/events/event_summary.csv.twig

```diff
@@ -0,0 +1,14 @@
+{% for metric in ['participants', 'pages-created', 'pages-improved', 'edits', 'byte-difference', 'files-uploaded', 'items-created', 'items-improved', 'pages-created-pageviews', 'pages-improved-pageviews-avg', 'pages-using-files', 'file-usage', 'pages-using-files-pageviews-avg', 'new-editors', 'retention'] %}{#
+    #}{% set stat = event.getStatistic(metric) %}{#
+    #}{% if stat is not empty %}{#
+        #}{{ msg(metric, [stat.offset]) }},{#
+        #}{% if stat.value is null %} {% else %}{% if metric == 'byte-difference' and stat.value > 0 %}+{% endif %}{{ stat.value }}{% endif %}
+
+{% endif %}{#
+#}{% endfor %}
+
+{{ msg('event-details') }}
+{{ msg('event-summary') }}, {{ event.displayTitle }}
+{{ msg('from') }}, {{ event.start|date_localize }}
+{{ msg('to') }}, {{ event.end|date_localize }}
+{{ msg('last-updated-table') }}, {{ event.updatedUTC|date_localize }}
```

Comment:
We are localizing the dates so we need to put the timezone somewhere. I think a new line is fine:
```
{{ msg('timezone') }}, {{ event.displayTimezone }}
```
Sorry I didn't include this in my example either.